src/components/Admin/AllDevices.vue

```diff
@@ -45,91 +45,261 @@
+                      >
+                        <v-text-field
+                          v-model="editedItem.deviceTag"
+                          label="Device Tag"
+                          color="black"
+                        />
+                      </v-col>
+                      <v-col
+                        cols="12"
+                        sm="6"
+                        md="4"
+                      >
+                        <v-text-field
+                          v-model="editedItem.deviceName"
+                          label="Name"
+                          color="black"
+                        />
+                      </v-col>
+                      <v-col
+                        cols="12"
+                        sm="6"
+                        md="4"
+                      >
+                        <v-text-field
+                          v-model="editedItem.status"
+                          label="Status"
+                          color="black"
+                        />
+                      </v-col>
+                    </v-row>
+                  </v-container>
+                </v-card-text>
+                <v-card-actions>
+                  <v-spacer />
+                  <v-btn
+                    color="#600000"
+                    text
+                    @click="close"
+                  >
+                    Cancel
+                  </v-btn>
+                  <v-btn
+                    color="#600000"
+                    text
+                    @click="save"
+                  >
+                    Save
+                  </v-btn>
+                </v-card-actions>
+              </v-card>
+            </v-dialog>
+          </v-toolbar>
+          <v-dialog
+            v-model="dialogDelete"
+            max-width="500px"
+          >
+            <v-card>
+              <v-card-title class="text-h5">
+                Are you sure you want to delete this item?
+              </v-card-title>
+              <v-card-actions>
+                <v-spacer />
+                <v-btn
+                  color="#600000"
+                  text
+                  @click="closeDelete"
+                >
+                  Cancel
+                </v-btn>
+                <v-btn
+                  color="#600000"
+                  text
+                  @click="deleteItemConfirm"
+                >
+                  OK
+                </v-btn>
+                <v-spacer />
+              </v-card-actions>
+            </v-card>
+          </v-dialog>
+        </template>
+        <template v-slot:[`item.actions`]="{ item }">
+          <v-icon
+            small
+            class="mr-2"
+            @click="editItem(item)"
+          >
+            mdi-pencil
+          </v-icon>
+          <v-icon
+            small
+            @click="deleteItem(item)"
+          >
+            mdi-delete
+          </v-icon>
+        </template>
+        <template v-slot:no-data>
```

Comment:
When you load the page, you can see the Reset button for a split second. Even if there is no data the reset button is just a bad idea. How does the admin know what "reset" does? Does it delete everything in the database? Does it refresh the page? They won't know these things. The average person just reloads a page if something doesn't work which achieves the same thing as your reset button. Also, if there's no data it's probably because there was an error, or the DB is truly empty and then we'd want to communicate that to them. I'd advise you to remove it entirely and as a team we can discuss what to display if there is nothing in the table.

Reply:
Besides that, everything else is good. Good job.

Reply:
I deleted that button. For right now, when the table loads, it will say no data (which is the default setting).